base/deprecated.jl

```diff
@@ -3115,7 +3119,7 @@ end
     @deprecate At_mul_B!(C::AbstractVector  , A::AbstractTriangular, B::AbstractVector)     mul!(C, Transpose(A), B)
     @deprecate At_mul_B!(C::AbstractMatrix  , A::AbstractTriangular, B::AbstractVecOrMat)   mul!(C, Transpose(A), B)
     @deprecate At_mul_B!(C::AbstractVecOrMat, A::AbstractTriangular, B::AbstractVecOrMat)   mul!(C, Transpose(A), B)
-    @deprecate A_mul_B!(A::Tridiagonal, B::AbstractTriangular)  mul!(A, B)
+    @deprecate A_mul_B!(A::Tridiagonal, B::AbstractTriangular)  mul2!(A, B)
```

Comment:
The odd method :).

Reply:
Agreed it is odd. We could get rid of it?